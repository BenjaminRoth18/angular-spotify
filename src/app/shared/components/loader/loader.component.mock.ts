import { Component } from '@angular/core';

@Component({
    selector: 'sp-loader',
    template: ``,
    styles: []
})
export class MockLoaderComponent {}
