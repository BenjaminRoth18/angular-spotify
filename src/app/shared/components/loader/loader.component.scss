.sp-loader {
    display: block;
    height: 100%;
    left: 0;
    position: fixed;
    top: 50%;
    transform: translateY(-50);
    width: 100%;
}
