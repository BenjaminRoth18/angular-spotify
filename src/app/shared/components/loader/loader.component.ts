import { Component } from '@angular/core';

@Component({
    selector: 'sp-loader',
    templateUrl: './loader.component.html',
    styleUrls: ['./loader.component.scss']
})
export class LoaderComponent {}
