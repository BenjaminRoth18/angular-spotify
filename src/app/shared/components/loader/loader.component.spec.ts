import { TestBed, ComponentFixture, async } from '@angular/core/testing';
import { LoaderComponent } from './loader.component';

describe('LoaderComponent', () => {
    let component: LoaderComponent;
    let fixture: ComponentFixture<LoaderComponent>;

    beforeEach(async(() => {
        TestBed.configureTestingModule({ declarations: [LoaderComponent] });

        fixture = TestBed.createComponent(LoaderComponent);
        component = fixture.debugElement.componentInstance;
    }));

    it('should create component', () => {
        expect(component).toBeTruthy();
    });

    describe('snaphots', () => {
        it('should create snapshot', () => {
            expect(fixture).toMatchSnapshot();
        });
    });
});
