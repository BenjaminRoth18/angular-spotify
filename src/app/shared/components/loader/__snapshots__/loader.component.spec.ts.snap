// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoaderComponent snaphots should create snapshot 1`] = `
<sp-loader>
  <div
    class="sp-loader d-flex justify-content-center"
  >
    <div
      class="spinner-border text-light"
      role="status"
    >
      <span
        class="sr-only"
      >
        Loading...
      </span>
    </div>
  </div>
</sp-loader>
`;
